### ISSUE TITLE
NPE in _LineChartState._handleBuiltInTouch

### ISSUE THREAD

**Issue by bakua** [imaNNeoFighT/fl_chart#651]
Hello, I got this error with version 0.35.0 on Flutter 2.0.5. I don't know how to replicate it.

```
Fatal Exception: io.flutter.plugins.firebase.crashlytics.FlutterError: Null check operator used on a null value.
       at State.setState(State.java:1287)
       at _LineChartState._handleBuiltInTouch(_LineChartState.java:88)
       at RenderLineChart.handleEvent(RenderLineChart.java:125)
       at GestureBinding.dispatchEvent(GestureBinding.java:361)
       at RendererBinding.dispatchEvent(RendererBinding.java:278)
       at GestureBinding._handlePointerEventImmediately(GestureBinding.java:316)
       at GestureBinding.handlePointerEvent(GestureBinding.java:280)
       at GestureBinding._flushPointerEventQueue(GestureBinding.java:238)
       at GestureBinding._handlePointerDataPacket(GestureBinding.java:221)
```

**Comment by imaNNeoFighT** [comment]
That's is very weird.
But I have some thoughts on it.
We will fix it ASAP.

**Comment by imaNNeoFighT** [comment]
Hi. I believe that it should be fixed in [0.36.1](https://github.com/imaNNeoFighT/fl_chart/releases/tag/0.36.1).

Can you check it out?

I will close this issue, and let me know if it doesn't fix in the future.

Thanks!
